py/selenium/webdriver/common/bidi/browser.py

```diff
@@ -123,16 +125,53 @@ def from_dict(cls, data: dict) -> "ClientWindowInfo":
         Returns:
         -------
             ClientWindowInfo: A new instance of ClientWindowInfo.
+
+        Raises:
+        ------
+            ValueError: If required fields are missing or have invalid types.
         """
-        return cls(
-            client_window=data.get("clientWindow"),
-            state=data.get("state"),
-            width=data.get("width"),
-            height=data.get("height"),
-            x=data.get("x"),
-            y=data.get("y"),
-            active=data.get("active"),
-        )
+        try:
+            client_window = data("clientWindow")
```

Comment:
```suggestion
            client_window = data["clientWindow"]
```
@pallavigitwork its good to run the bazel tests locally first before pushing. You can run it using - `bazel test //py:common-firefox-bidi-test/selenium/webdriver/common/bidi_browser_tests.py` for this test.

Reply:
The CI is failing because of above.

Reply:
sorry, i am trying AI help to fix issues. Didn't notice that Claude uses ( instead of [. Thank you for the changes. I will notice the brackets used by AI next time as well. Understanding code written by someone else is quite tedious.. im finding lot of help with AI, and these fixes are helping me learn as well how Selenium and these BiDi is working, and is being implemented. so thank you raising the bug Corey - 15697. thanks for correction Navin.

Reply:
Sure, no worries. We are here to help you navigate the codebase!

Reply:
thank you for your kind help and time.